app/src/routes/v1/team.router.js

```diff
@@ -1,6 +1,6 @@
 const Router = require("koa-router");
 const logger = require("logger");
-const TeamModel = require("models/team.model");
+const TeamModel = require("models/team.model").default;
```

Comment:
Team model is now an ES Module, need .default for it to still work in old commonJS files!